Wave1_data/1.preprocess_data/scripts/0.preprocess_raw_data.py

```diff
@@ -171,10 +239,10 @@
 df[["well", "FOV"]].value_counts().sort_values()
 
 
-# In[10]:
+# In[15]:
 
 
-# get all files in the 20241025T045229_ dir
+# get all files in the 20241026T164425_ dir
```

Comment:
Leading back to my plate question, are you only processing one of these plates only so plate won't matter?